test/clj/rems/test/api/applications.clj

```diff
@@ -924,9 +924,7 @@
           (is (= ["application.event/created"
                   "application.event/draft-saved"
                   "application.event/submitted"
-                  ; decision- and comment-requested, and decided should be missing here
-                  "application.event/commented"
-                  "application.event/commented"
+                  ; all decision and comment related events should be missing here
```

Comment:
Should this test be rewritten so that both events that can be seen and cannot be seen are explicitly listed? There have been bugs related to this functionality before.